components/src/molecules/logo/Logo.stories.js

```diff
@@ -0,0 +1,36 @@
+import { addDecorator, storiesOf } from '@storybook/vue'
+import StoryRouter from 'storybook-vue-router'
+
+import ALogo from './Logo.vue'
+
+addDecorator(StoryRouter())
```

Comment:
can you add decorator under storiesOff?

Reply:
Did you mean this way? 

`storiesOf('Modules/Side Menu', module)
  .addDecorator(StoryRouter())`

Firstly I tried do it, but than we have a problem with order of imports , and get a errors. We can add it globally in config, and it will work, but I don't know if this is a good way.

Reply:
ok so keep it in this way